modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableViewSkin.java

```diff
@@ -136,7 +137,6 @@ public TableViewSkin(final TableView<T> control) {
     }
 
 
-
```

Comment:
There are no changes in this file other than white space and unused imports, so this file can be reverted.